jax/BUILD

```diff
@@ -401,9 +401,11 @@ pytype_strict_library(
         ":compilation_cache_interface",
         ":config",
         ":gfile_cache",
+        ":lru_cache",
         ":monitoring",
         ":path",
         "//jax/_src/lib",
+        "//third_party/py/filelock",
```

Comment:
Add that to `lru_cache` deps?

You will want to use the `py_deps` macros for this: `py_deps("filelock")`. We have a list of deps in jax.bzl.